backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java

```diff
@@ -1150,6 +1150,41 @@ public WorkspaceDTO getWorkspaceDTOByWorkspaceId(
     }
   }
 
+  private VisibilityEnum.Visibility fromResourceVisibility(Integer workspaceType, ResourceVisibility resourceVisibility) {
+    if (workspaceType == WorkspaceType.ORGANIZATION.getNumber()) {
+      switch(resourceVisibility) {
+        case PRIVATE:
+          return VisibilityEnum.Visibility.PRIVATE;
+        case ORG_DEFAULT:
```

Comment:
@conradoverta what should this be?  I can't figure out how to convert this.

Reply:
Why are we doing the conversion from the resource visibility to the project visibility?

Reply:
UAC returns a ResourceVisibility, but the Project has a VisibilityEnum.Visibility

Reply:
I'd recommend changing the project representation for full compatibility. Otherwise we won't be able to represent the full range of possibilities. And yes, I'm aware this is a breaking change. Need to check the impact on the client.